README.md

```diff
@@ -746,24 +879,44 @@ int worker_main() {
 }
 ```
 
+> 📕 Reference documentation for:
+> * [initialization and shutdown](https://dynatrace.github.io/OneAgent-SDK-for-C/group__init.html)
+> * [`onesdk_agent_get_fork_state`](https://dynatrace.github.io/OneAgent-SDK-for-C/group__misc.html#ga77260efaf63455969962e05b6b170135)
+
+[refd_initialize_2]: https://dynatrace.github.io/OneAgent-SDK-for-C/group__init.html#gac0681af704ba7e6404c3f67f582ee4db
+[refd_init_flag_forkable]: https://dynatrace.github.io/OneAgent-SDK-for-C/group__init.html#ga732bf07f0e190264baf29f3a1c22cc4a
+
+
+<a name="troubleshooting"></a>
 
 ## Troubleshooting
 
-If the SDK stub cannot load or initialize the agent module (see output of sample1), you can set the SDK stub's logging level to activate logging by either
-- calling `onesdk_stub_set_logging_level(ONESDK_LOGGING_LEVEL_{LEVEL})`
-- setting the environment variable `DT_LOGLEVELSDK={level}`
-- if your program passes command line arguments to the SDK (see `onesdk_stub_process_cmdline_args`), you can use the command line argument `--dt_loglevelsdk={level}`
+If the SDK stub cannot load or initialize the agent module (see output of sample1), you can set the SDK stub's logging level to activate
+logging by either
+- calling [`onesdk_stub_set_logging_level(ONESDK_LOGGING_LEVEL_{LEVEL})`](refd_stub_set_logging_level]; or
```

Comment:
The "refd_..." link syntax here is incorrect - I guess it needs to be `[foo]` instead of `(foo]`.

Reply:
Fixed.